### PR TITLE
Add isReplicated

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/590/HighFidelity-Beta-PR10666-84e1a6f89388d797492fea97b8f966156a48a394.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/590/HighFidelity-Beta-PR10666-84e1a6f89388d797492fea97b8f966156a48a394.exe

File: libraries/networking/src/NodeList.cpp [hunk 1]
```diff
@@ -654,8 +654,11 @@ void NodeList::parseNodeFromPacketStream(QDataStream& packetStream) {
     QUuid nodeUUID, connectionUUID;
     HifiSockAddr nodePublicSocket, nodeLocalSocket;
     NodePermissions permissions;
+    bool isReplicated;
 
-    packetStream >> nodeType >> nodeUUID >> nodePublicSocket >> nodeLocalSocket >> permissions;
+    packetStream >> nodeType >> nodeUUID >> nodePublicSocket >> nodeLocalSocket >> permissions >> isReplicated;
+
+    qDebug() << "Node: " << nodeUUID << nodeType << isReplicated;
```
Comment (birarda) [inline libraries/networking/src/NodeList.cpp]: Do we need this?

Comment (huffman) [inline libraries/networking/src/NodeList.cpp]: will remove

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/601/HighFidelity-Beta-PR10666-4851d3b56409bef465365a552858ea8746992ade.exe

File: libraries/networking/src/LimitedNodeList.cpp [hunk 2]
```diff
@@ -580,11 +581,12 @@ SharedNodePointer LimitedNodeList::addOrUpdateNode(const QUuid& uuid, NodeType_t
         matchingNode->setLocalSocket(localSocket);
         matchingNode->setPermissions(permissions);
         matchingNode->setConnectionSecret(connectionSecret);
+        matchingNode->setIsReplicated(isReplicated);
 
         return matchingNode;
     } else {
         // we didn't have this node, so add them
-        Node* newNode = new Node(uuid, nodeType, publicSocket, localSocket, permissions, connectionSecret);
+        Node* newNode = new Node(uuid, nodeType, publicSocket, localSocket, permissions, isReplicated, connectionSecret, this);
```
Comment (Atlante45) [inline libraries/networking/src/LimitedNodeList.cpp]: Do you need to add the parenting back?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/601/HighFidelity-Beta-PR10666-4851d3b56409bef465365a552858ea8746992ade.dmg

**Pull request closed without merging**